install/kubernetes/helm/istio/values.yaml

```diff
@@ -440,6 +440,22 @@ global:
   #
   meshNetworks: {}
 
+  # Specifies the global locality load balancing settings. One of failover or distribute can be set.
+  # localityLbSetting:
+  #   distribute:
+  #   - from: "us-central1/*"
+  #     to:
+  #       "us-central1/*": 80
+  #       "us-central2/*": 20
+  #
+  # localityLbSetting:
+  #   failover:
+  #   - from: us-east
```

Comment:
why is from and to syntax different between distribute and failover?   "us-central1/*" vs "us-central1"

Reply:
i think it is because you can specify zone info in distribute but not in failover (which is more region).

Reply:
Yes, failover is just region, distribute is region/zone/subzone